articles/azure/asz-sco.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: UKCloud for Microsoft Azure Service Scope | UKCloud Ltd
+title: UKCloud for Microsoft Azure service scope | UKCloud Ltd
```

Comment:
Actually, we use Init Caps for article types, so this should stay as Service Scope (sorry, should have included that in my list of exceptions)